internal/github/checkPendingCI.go

```diff
@@ -0,0 +1,66 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/google/go-github/v60/github" // Ensure your go-github library version matches
+
+	"github.com/chia-network/github-bot/internal/config"
+)
+
+// CheckForPendingCI will return a list of PR URLs that are ready for CI to run but haven't started yet.
+func CheckForPendingCI(githubClient *github.Client, internalTeam string, cfg config.LabelConfig) ([]string, error) {
+	teamMembers, _ := GetTeamMemberList(githubClient, internalTeam)
+	var pendingPRs []string
+	for _, fullRepo := range cfg.LabelCheckRepos {
```

Comment:
`cfg.LabelCheckRepos` -- should this have its own repo list actually? It's a different check than the label checks, which has me leaning towards "probably?"

Reply:
I added an additional config section for listing repos that will checked for stale/pending PRs